src/utils/environment.ts

```diff
@@ -27,8 +28,8 @@ export function getRequiredEnvVar(varName: string): string | never {
  * @param type
  * @returns {string}
  */
-export function readFile(path: string, type: string): string {
+export const readFile = (path: string, type: string): string => {
   log.info("Reading %s file from %s", type, path);
-  // eslint-disable-next-line security/detect-non-literal-fs-filename
+  // eslint-disable-next-line
```

Comment:
This line seems not required

Reply:
@michaeldisaro in general, avoid disabling eslint totally - always be specific on which rule you're going to disable

Reply:
ok, I fixed that behavior on my tool.

Reply:
I removed the line, it was effectively useless.